src/o365/tenant/commands/id/id-get.ts

```diff
@@ -65,23 +72,20 @@ class TenantIdGetCommand extends Command {
     return options.concat(parentOptions);
```

Comment:
Since `domain` is now optional, it should be indicated as such via `[domainName]`